CMakeLists.txt

```diff
@@ -203,6 +203,7 @@ include_directories(src/include)
 include_directories(third_party/antlr4_cypher/include)
 include_directories(third_party/antlr4_runtime/src)
 include_directories(third_party/fast_float/include)
+include_directories(third_party/mbedtls/include)
 include_directories(third_party/miniparquet/src)
```

Comment:
httpfs extension has already imported the mbedlts library, can you remove the mbedlts library from httpfs extension?

Reply:
I've already moved the mbedtls library from httpfs to third_party folder. Could you check these 2 commits?
https://github.com/kuzudb/kuzu/pull/3212/commits/e6a857e9171dbb4e33d8a4986d1af3e07fefb743
https://github.com/kuzudb/kuzu/pull/3212/commits/c05addc34757decbe55e0d3558a689863dd3b9f5

Reply:
I see, good job